.github/ISSUE_TEMPLATE/rules-decoders_request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Ruleset issue
+about: Report a bug or make a feature request.
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
Are these lines needed?

Reply:
```suggestion

```
Not really, it was inserted by error when copying the table below.